openassessment/templates/openassessmentblock/response/oa_response.html

```diff
@@ -74,7 +74,11 @@ <h4 class="step__title">
                                 <h5 class="submission__answer__part__text__title">{% trans "The prompt for this section" %}</h5>
                                 <article class="submission__answer__part__prompt">
                                     <div class="submission__answer__part__prompt__copy">
-                                        {{ part.prompt.description|linebreaks }}
+                                        {% if prompts_type == 'html' %}
+                                            {% autoescape off %}{{ part.prompt.description }}{% endautoescape %}
```

Comment:
Same comment as above: This might be cleaner with:
```{{ part.prompt.description|safe }}```
You don't need the autoesape off and on

Reply:
fixed